compiler/rustc_const_eval/src/const_eval/target.rs

```diff
@@ -0,0 +1,656 @@
+use rustc_hir::def::DefKind;
```

Comment:
What happened here, why is there such a new file?

Reply:
huh... that was in my edits during branch switching and probably got saved by vscode after the branch switch removed it